src/partials/products-section.html

```diff
@@ -4,28 +4,43 @@
     <h2 class="products__title">PRODUCTS</h2>
     <ul class="products__list">
       <li class="products__list-item">
-        <h3>ice cream</h3>
-        <div>...</div>
-        <p>
+        <h3 class="products__subtitle">ice cream</h3>
+        <div class="products__suspension-dots"></div>
+        <p class="products__description">
           Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do
           eiusmod tempor incididunt ut labore et
         </p>
+        <button class="products__btn" type="button">
+          <svg width="40px" height="40px" fill="#D41443">
+            <use href="/src/images/sprite.svg#arrow-right"></use>
+          </svg>
+        </button>
       </li>
       <li class="products__list-item">
-        <h3>ice coffee</h3>
-        <div>...</div>
-        <p>
+        <h3 class="products__subtitle">ice coffee</h3>
+        <div class="products__suspension-dots"></div>
+        <p class="products__description">
           Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do
           eiusmod tempor incididunt ut labore et
         </p>
+        <button class="products__btn" type="button">
+          <svg width="40px" height="40px" fill="#D41443">
+            <use href="/src/images/sprite.svg#arrow-right"></use>
+          </svg>
+        </button>
       </li>
       <li class="products__list-item">
-        <h3>milkshakes</h3>
-        <div>...</div>
-        <p>
+        <h3 class="products__subtitle">milkshakes</h3>
+        <div class="products__suspension-dots"></div>
+        <p class="products__description">
           Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do
           eiusmod tempor incididunt ut labore et
         </p>
+        <button class="products__btn" type="button">
+          <svg class="products__svg" width="40px" height="40px" fill="#D41443">
```

Comment:
svg-ul daca apartibe butonului este arrow-button__svg; in cazul in care ar fi fost doar in sesctiunea ta era products__btn-svg

Reply:
am modificat, am pus pe common clasele cu style pentru buttonul cu svg si eu miam pus modificator sa pot sal pozitionex si am luat cu extend ce avem nevoie